docs/en/latest/plugins/openid-connect.md

```diff
@@ -33,29 +33,29 @@ The `openid-connect` Plugin provides authentication and introspection capability
+| client_id                            | string  | True     |                       |              | OAuth client ID.                                                                                                         |
+| client_secret                        | string  | True     |                       |              | OAuth client secret.                                                                                                     |
+| discovery                            | string  | True     |                       |              | Discovery endpoint URL of the identity server.                                                                           |
+| scope                                | string  | False    | "openid"              |              | Scope used for authentication.                                                                                           |
+| realm                                | string  | False    | "apisix"              |              | Realm used for authentication.                                                                                           |
+| bearer_only                          | boolean | False    | false                 |              | When set to true, the Plugin will check for if the authorization header in the request matches a bearer token.           |
+| logout_path                          | string  | False    | "/logout"             |              | Path for logging out.                                                                                                    |
+| post_logout_redirect_uri             | string  | False    |                       |              | URL to redirect to after logging out.                                                                                    |
+| redirect_uri                         | string  | False    | "ngx.var.request_uri" |              | URI to which the identity provider redirects back to.                                                                    |
+| timeout                              | integer | False    | 3                     | [1,...]      | Request timeout time in seconds.                                                                                         |
+| ssl_verify                           | boolean | False    | false                 |              | When set to true, verifies the identity provider's SSL certificates.                                                     |
+| introspection_endpoint               | string  | False    |                       |              | URL of the token verification endpoint of the identity server.                                                           |
+| introspection_endpoint_auth_method   | string  | False    |                       |              | Authentication method name for token introspection, If not specified, the default will use the first value in well_know. |
```

Comment:
I'm not clear, cc @starsz to have a check 🙏

Reply:
Ok.Let me have a check.

Reply:
After digging it into.I found that:

1. The default value of `introspection_endpoint_auth_method` is "client_secret_basic", sorry to @hf400159 .
2. And the `lua-resty-openidc` will use another conf called "token_endpoint_auth_method".It doesn't introduce in the docs, but it can work well in "openid-connection" plugin.

The default value of "token_endpoint_auth_method" will fetch the first supported method specified by the OP.
refer https://github.com/zmartzone/lua-resty-openidc/blob/master/lib/resty/openidc.lua#L677

So in conclusion, I think we should keep the original doc, and add the new filed of "token_endpoint_auth_method" to the docs.

Reply:
@starsz fixed. PTAL 😄